src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -84,11 +84,13 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             // as we may have installed a missing package.
 
             DependencyView.CustomNodeManager = (CustomNodeManager)viewLoadedParams.StartupParams.CustomNodeManager;
-
-            pmExtension.PackageLoader.PackgeLoaded += (package) =>
+            if (pmExtension != null)
```

Comment:
If pmExtenions does not exist does it makes sense to load this one or we can simply return here at the beginning ?

Reply:
There are other types of workspace dependencies (e.g. local dependency, custom node workspace dependency) that are not depending on pmExtension, so it still make sense to load I believe.